### ISSUE TITLE
[FEATURE] Реализовать систему отчетности

### ISSUE THREAD

**Issue by dzeron13** [alor-broker/Astras-Trading-UI#185]
### Опишите проблему, которую решает ваша фича

На текущий момент вести какую-то отчетность по сделкам очень трудно. Тот формат, что представлен в ЛК - это, прошу прощения "обнять и плакать". Там представлены отчет за прошлый месяц и по дням, опять же максимум за предыдущий.
Поэтому в конце торгового дня нельзя понять, был ли день успешен, только вести подсчет сделок вручную.
Сейчас приходится пользоваться сторонними сервисами, где по API это всё получают и красиво выводят.

### Опишите предлагаемое вами решение

Реализовать систему отчетности с возможностью оперативно видеть последние сделки.
Понимаю, что задача, возможно, требует каких-то глобальных изменений бекэнда, но пусть будет на будущее.
Из того, что требуется торговцам:
- список сделок за период, что-то вроде: "дата, время, код операции, бумага, вид операции, валюта, комиссия" и т.п. Просто список в csv/xls. Многие ведут аналитику самостоятельно и им нужен банальный список операций.
- отчет по одному инструменту за период. Т.е. поторговал бумагу за день, посмотрел сколько прибыль и комиссия. Вот пример из сторонней разработки:
![image](https://user-images.githubusercontent.com/99450543/169490539-0d9163bf-25a0-43fc-9570-420f66dabb35.png)
- финансовый результат по инструменту (списку) за период, как предыдущий пункт, но списком и в виде отчета: "инструмент, цена входа-выхода, комиссия, результат". Что-то такое:
![image](https://user-images.githubusercontent.com/99450543/169492888-ba988000-fdd8-4aa9-8b66-81e3db7bb50f.png)
Плюс, с возможностью расширенной аналитики, т.е. кроме итоговой суммы - все сделки по инструменту. Вот тоже для примера:
![image](https://user-images.githubusercontent.com/99450543/169512398-9cff66ae-0683-450c-bc0b-6aca8ffcd0ce.png)

Чтобы не переходить лишний раз в ЛК, реализацию предлагаю сделать в самом терминале, допустим вкладкой в Профиле или может быть отдельным виджетом.

### Правила

- [ ] Я согласен следовать правилам

**Comment by PilgrimViis** [comment]
Да, вы правы. Отчёты сейчас негодятся. Но тут есть большие технические и организационные сложности. Я передал коллегам, которые отвечают за это. Но честно говоря, не думаю что это сделают в ближайший год точно =(( Мы подумаем, может быть сможем как-то в рамках самого терминала улучишь показ этой информации.
